### ISSUE TITLE
Cannot start waydroid Session

### ISSUE THREAD

**Issue by connorwilliams1** [waydroid/waydroid#988]
**Describe the bug**
I have installed, removed, and reinstalled multiple times in an attempt to troubleshoot.  I am unable to start a waydroid session with the following error:

[09:15:50] WAYLAND_DISPLAY is not set, defaulting to "wayland-0"
[09:15:50] Wayland socket '/run/user/1000/wayland-0' doesn't exist; are you running a Wayland compositor?

**Screenshots**
[https://i.imgur.com/Xc62keZ.png](url)

**General information (please complete the following information):**
 - Waydroid tools Version - 1.4.1
 - Waydroid Images Version [e.g. VANILLA 20211021], or `system_datetime`/`vendor_datetime` from `/var/lib/waydroid/waydroid.cfg`  - the file listed displays "0" next to system_datetime and vendor_datetime

**Desktop (please complete the following information):**
 - OS: Pop_OS! 22.04 LTS x86_64
 - GPU: NVIDIA Geforce GTX 1080
 - Kernel version: 6.2.6-76060206-generic
 - Host mesa version: 1.3.211 (I think, might need help finding this if thats not correct, I utilized vulkaninfo --summary to find this)
 - Desktop: GNOME 42.5

I'm not totally new to utilizing linux but I'm not an expert, either, so I apologize in advance if I'm missing something obvious.

**Logs  
[dmesg.log](https://github.com/waydroid/waydroid/files/11916799/dmesg.log)
[waydroid.log](https://github.com/waydroid/waydroid/files/11916800/waydroid.log)
[waydroid_base.txt](https://github.com/waydroid/waydroid/files/11916803/waydroid_base.txt)

**Comment by mr-sihc** [comment]
instead of opens issue, seach for "wayland GNOME" in internet
and use wayland gnome.

**Comment by n1lby73** [comment]
you can also try using nvidia open source driver instead of proprietry driver to see if  wayland would function properly

**Comment by Kanhoto** [comment]
is there nothing to do about it ? I tried switching my drivers from Nvidia to xorg, but it still doesn't show UI and I have the exact same error.

**Comment by gander** [comment]
> instead of open issue, seach for "wayland GNOME POP OS" in internet and use wayland gnome.

How I love useless answers like this. THIS is the right place for SUCH questions and HERE should be answered by the developers or community.

**Comment by gander** [comment]
Unfortunately, I currently have Linux Mint 21.2 and do not have support for Wayland. I have to wait for the update until 21.3, when [experimental support for Wayland](https://forums.linuxmint.com/viewtopic.php?t=380435) will appear.

**Comment by gander** [comment]
@ZelenaHull Yes, it helped, but not everything wanted to work yet, so I found some helpful script [here](https://www.reddit.com/r/linuxmint/comments/16680tz/comment/jyia750/)

**Comment by gander** [comment]
I'm currently stuck here. Sometimes I see the clock for a split second before it fades to gray. I can't click anything except launch the console in the top left corner.

[stuck-here.webm](https://github.com/waydroid/waydroid/assets/421597/b186de9e-7fbc-4e62-ba46-f4065de71a73)

**Comment by ET1Veg** [comment]
Found this site helpful also: https://community.kde.org/Plasma/Wayland
